### PR TITLE
Feat/#1426 program services modal

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

Add program services modal

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: global/config.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ export const getConfig = () => {
       publicConfig.EGO_API_ROOT || '',
       `/api/oauth/login/google?client_id=${publicConfig.EGO_CLIENT_ID || ''}`,
     ),
-    DOCS_URL_ROOT: publicConfig.DOCS_URL_ROOT || 'https://docs.icgc-argo.org/docs/',
     DACO_URL: publicConfig.DACO_URL || 'https://icgc.org/daco',
+    DOCS_URL_ROOT: publicConfig.DOCS_URL_ROOT || 'https://docs.icgc-argo.org',
```
Comment (hlminh2000) [inline global/config.ts:21]: this looks ok to change for me, but wanna confirm... is anything existing affected by this?

File: components/pages/Homepage/index.tsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ import {
   ResponsiveGridLayout,
 } from './common';
 import { FILE_REPOSITORY_PATH } from 'global/constants/pages';
+import { ModalPortal } from 'components/ApplicationRoot';
+import ProgramServicesModal from './ProgramServicesModal';
```
Comment (hlminh2000) [inline components/pages/Homepage/index.tsx:23]: these aren't used here eh?

File: components/NavBar.tsx [hunk 3]
```diff
@@ -154,11 +163,12 @@ export default function Navbar({ hideLink = false, disableLogoLink = false }) {
...
-                    <Typography variant={'default'}>Submission</Typography>
-                  </MenuItem>
+                  {' '}
+                  {programServices}
                 </a>
               </Link>
+            ) : (
+              <div onClick={() => setModalVisibility(!isModalVisible)}> {programServices}</div>
             )}
             {!userModel && (
```
Comment (anncatton) [inline components/NavBar.tsx:173]: I think the login menu item needs some extra margin now that program services is showing for logged out state

File: global/config.ts (hunk 1 above)
Comment (ciaranschutte) [reply global/config.ts:21]: I don't think so. 
Docs pages are linked here: https://github.com/icgc-argo/platform-ui/blob/develop/global/constants/docSitePaths.tsx with the `/docs/` prefix in their strings

The root for the text documentation, not dictionary, does not have `/docs/` in it's path @hlminh2000

File: components/NavBar.tsx (hunk 3 above)
Comment (ciaranschutte) [reply components/NavBar.tsx:173]: @anncatton nice catch - thank you! Added horizontal padding to login button menu item

**Pull request merged**